EXE/dacomsrv.ini

```diff
@@ -0,0 +1,31 @@
+@include FL_Dev.ini
+
+[DACOM]
+IgnoreDACOMEnv = true
+DllPath = .\
+
+[Libraries]
+readfile.dll
+engbase.dll
+x86math.dll
+rendcomp.dll
+system.dll
+Thorn.dll
+DebugLib.dll
+rp8.dll
+shading.dll
+FLHook.dll
```

Comment:
Do we want to enable FLHook by default? I thought the idea was to add an option to the installer for that since it's dependent on the VC++ runtime libraries.

Reply:
Yeah an option with a link to the runtime is probably a good idea. Or I don't know if Inno can open up the runtime installer on completion? Not sure what the capabilities are.

Reply:
Yes, that's possible. The MSVC redistributable package can essentially be added as an entry to `[Files]`, which can then be executed whenever necessary via `[Run]`. We have to do this anyway since the mod has a few features that are also dependent on some VC++ runtime libraries, see https://github.com/oliverpechey/Freelancer-hd-edition-install-script/issues/110.